src/main/java/teammates/ui/webapi/CreateAccountAction.java

```diff
@@ -94,14 +100,18 @@ public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOpera
 
         try {
             logic.updateAccountRequest(AccountRequestAttributes
-                    .updateOptionsBuilder(instructorEmail, instructorInstitution)
+                    .updateOptionsBuilder(instructorEmail, instructorInstitute)
+                    .withStatus(AccountRequestStatus.REGISTERED)
                     .withRegisteredAt(Instant.now())
                     .build());
         } catch (EntityDoesNotExistException | InvalidParametersException e) {
             // EntityDoesNotExistException should not be thrown as existence of account request has been validated before.
             // InvalidParametersException should not be thrown as there should not be any invalid parameters.
             log.severe("Unexpected error", e);
             return new JsonResult(e.getMessage(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
+        } catch (EntityAlreadyExistsException e) {
+            log.severe("The account request is not supposed to be re-created.", e);
+            return new JsonResult(e.getMessage(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
This should be combined with the previous case.

Reply:
I'm fine with that.